src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource_Async.cs

```diff
@@ -110,17 +111,22 @@ private partial class SuggestedActionsSource : IAsyncSuggestedActionsSource
                     // then pass it continuously from one priority group to the next.
                     var lowPriorityAnalyzers = new ConcurrentSet<DiagnosticAnalyzer>();
 
+                    using var _2 = TelemetryLogging.LogBlockTimeAggregated(FunctionId.SuggestedAction_Summary, "Total");
```

Comment:
what's the diff between this and SuggestedAction_Application_Summary?  Is it so we can measure what time is spent in the source, prior to actually doing computation at each pri level?

Reply:
SuggestedAction_Summary is intended to measure the time to calculate the actions at each priority, whereas SuggestedAction_Application_Summary is intended to measure the time to apply the changes for an invoked suggested action.